src/panel/Menu.jsx

```diff
@@ -36,58 +37,47 @@ const Menu = () => {
   };
 
   return <Fragment>
-    <MenuButton onClick={open} />
+    <button
+      type="button"
+      className={cx('menu__button', { 'menu__button--active': isOpen })}
+      onClick={toggle}
+      title={_('Menu')}
+    >
+      <IconMenu />
+    </button>
 
     {isOpen && ReactDOM.createPortal(<div className="menu">
       <div className="menu__overlay" onClick={close} />
 
       <div className="menu__panel">
-        <div className="menu__panel__top">
-          <h2 className="menu__panel__top__title">
-            <i className="menu__panel__top__icon icon-map" />
-            <span>Qwant Maps</span>
-            <i className="icon-x menu__panel__top__close" onClick={close} />
-          </h2>
-        </div>
-
-        <div className="menu__panel__items_container">
-          <div className="menu__panel__section menu__panel__section-internal">
-            <button className="menu__panel__action"
-              onClick={() => { navTo('/', { focusSearch: true }); }}
-            >
-              <img
-                className="menu__panel__action__icon"
-                src="./statics/images/magnifier-dark.svg" alt=""
-              />
-              <span>{_('Search', 'menu')}</span>
-            </button>
-            {isDirectionActive &&
-              <button className="menu__panel__action" onClick={() => {
-                Telemetry.add(Telemetry.MENU_ITINERARY);
```

Comment:
if we get rid of this specific event, we can remove the `menu_itinerary` event from the `telemetry` module